contrib/qubes/misc/0017-esrt-Add-paravirtualization-support.patch

```diff
@@ -0,0 +1,65 @@
+From 2b3e4dcc1dceda2a7bca3dbe36165ab7b1e9bd51 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Norbert=20Kami=C5=84ski?= <norbert.kaminski@3mdeb.com>
+Date: Mon, 17 Aug 2020 06:04:05 -0400
+Subject: [PATCH] esrt: Add paravirtualization support
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Signed-off-by: Norbert Kamiński <norbert.kaminski@3mdeb.com>
```

Comment:
Before we start carrying this in tree - has it been sent up to the kernel mailing list?  Was it rejected?  Or is it in newer kernels already now?  I think it's better that we refer to an LKML link than a .patch file carried in the fwupd tree.

Reply:
It wasn't upstreamed, because it is Xen related problem.  [The Xen patch](https://patchwork.kernel.org/project/xen-devel/patch/20200816001949.595424-1-marmarek@invisiblethingslab.com/) was upstreamed but it stuck some time ago. I asked @marmarek if this patch could be added to the [qubes-linux-kernel](https://github.com/QubesOS/qubes-linux-kernel) repo.